json/src/spec/params.rs

```diff
@@ -152,6 +152,9 @@ pub struct Params {
 	/// KIP4 activiation block height.
 	#[serde(rename="kip4Transition")]
 	pub kip4_transition: Option<Uint>,
+	/// Wasm `gasleft` extern (aka opcode) activation block height.
+	#[serde(rename="wasmGasleftActivationTransition")]
+	pub wasm_gasleft_activation_transition: Option<Uint>,
```

Comment:
i wonder if we can bind it to kip4_transition

how about extending kip4, since it is not activated so far

@debris @sorpaas  ?

Reply:
Agree that we can use 1 switch for that. However, I think we should update the KIP4 proposal itself. Otherwise, it might be very confusing

Reply:
Yeah, I will add another kip then

Reply:
so now it is 2 switches, but they can be activated at once (with the same block number)

please review @pepyakin @sorpaas